keras_vggface/vggface.py

```diff
@@ -65,37 +65,6 @@ def VGGFace(include_top=True, model='vgg16', weights='vggface',
                          '`None` (random initialization) or `vggface`'
                          '(pre-training on VGGFace Datasets).')
 
```

Comment:
I removed models and flexibility in the SeNet model as I was debugging some issues and using TensorFlow instead of keras. It also made it clearer for me to understand. It's easy for me to put them back as I now understand, and can put back the other models.

Reply:
However, I also removed logic in SeNet that we didn't use, adding it back in may complicate the code for future readers.

Reply:
I have added back the models, as I did not affect them.